### ISSUE TITLE
The accuracy of the provided pre-trained model for the mini-imagenet does not match the paper

### ISSUE THREAD

**Issue by CapableCat** [solangii/MICS#2]
I appreciate your work very much.

During the reproduction process, I found that the pre-trained model you provided for mini-imagent has an accuracy of 74.68%, which is significantly lower than the 84.40% reported in the paper. I am confused about this, can you please tell me if the pre-trained model you provided for mini-imagent matches the paper?

The reproduction results on mini-imagenet using the provided pre-trained model are as follows:
           sess1       sess2      sess3       sess4       sess5     sess6     sess7      sess8      sess9
Acc:     74.68%   69.49%   64.81%   61.56%   59.08%   56.29%   53.9%    52.44%   51.3%
Base:    74.68%   74.3%    68.97%   64.57%   61.2%    58.58%   55.88%   53.66%   52.02% 
Novel:   0.0%     11.8%    10.8%    19.4%    27.2%    19.8%    20.2%    30.6%    37.6%    
Old:      74.68%   74.3%    73.83%   73.58%   73.22%   72.93%   72.42%   72.1%    71.75%   
New:     0.0%     11.8%    10.7%    13.47%   16.65%   16.36%   16.87%   18.74%   20.63%
